packages/explorer/src/components/playground/CreateNew.tsx

```diff
@@ -108,10 +111,12 @@ const CreateNew = (props: {
         const { res } = await computer.sync(txId)
         setFunctionResult({ _rev: res._rev, type: "objects" })
         setModalTitle("Success!")
+        showLoader(false)
```

Comment:
As above if possible it would be better to move the showLoader(false) to below the try catch block